src/js/hopscotch.js

```diff
@@ -2430,8 +2431,90 @@
 // Template includes, placed inside a closure to ensure we don't
 // end up declaring our shim globally.
 (function(){
-// @@include('../../src/tl/_template_headers.js') //
-// @@include('../../tmp/js/hopscotch_templates.js') //
+var _ = {};
```

Comment:
This shouldn't be included in /src. This code gets added in when running `grunt`. If you need the output from `grunt` for testing purposes, the built files will be placed in the `tmp/` folder.